safeeyes/plugins/smartpause/plugin.py

```diff
@@ -194,7 +194,8 @@ def __start_idle_monitor():
                 smart_pause_activated = True
                 idle_start_time = datetime.datetime.now() - datetime.timedelta(seconds=system_idle_time)
                 logging.info('Pause Safe Eyes due to system idle')
-                disable_safeeyes(None, True)
+                info = _('Paused Safe Eyes due to system being idle')
+                self.disable_safeeyes(info)
```

Comment:
This does not fix the issue, now smartpause fails with `NameError: name 'self' is not defined`.